build/mac/notarize_dmg_pkg.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
was this copied from chrome? It looks like it was and I'm curious why, but also we can't copy a chromium file and use our own copyright. That's a violation of their copyright

Reply:
No, it merges the `brave/script/signing_helper.py` create_config section with the notarization calls from `chrome/installer/mac/signing/pipeline.py`, but otherwise it is not copied from Chromium.

Reply:
if it includes any copied chromium code we should include their copyright notice as well

Reply:
@bridiver only a few lines are similar to the Chromium one, the rest is all ours- it's def not copied. Looks good to me 👍 

We could reference the area which has similarities and say something like "Logic based on Chromium one at `src/components/blah/installer/`" to attribute it

Reply:
this file is also in the wrong place. Scripts should go in the scripts directory